### ISSUE TITLE
ValueError: No JSON object could be decoded in Selenium-py 2.49.0

### ISSUE THREAD

**Issue by nikolas** [SeleniumHQ/selenium#1478]
I've updated from selenium 2.48.0 to 2.49.0, and I'm getting this error:

```
Traceback (most recent call last):
        File "/home/travis/build/ccnmtl/worth2/ve/lib/python2.7/site-packages/behave/model.py", line 1456, in run
          match.run(runner.context)
        File "/home/travis/build/ccnmtl/worth2/ve/lib/python2.7/site-packages/behave/model.py", line 1903, in run
          self.func(context, *args, **kwargs)
        File "features/steps/common.py", line 67, in i_click_the_link
          element.click()
        File "/home/travis/build/ccnmtl/worth2/ve/lib/python2.7/site-packages/selenium/webdriver/remote/webelement.py", line 75, in click
          self._execute(Command.CLICK_ELEMENT)
        File "/home/travis/build/ccnmtl/worth2/ve/lib/python2.7/site-packages/selenium/webdriver/remote/webelement.py", line 454, in _execute
          return self._parent.execute(command, params)
        File "/home/travis/build/ccnmtl/worth2/ve/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 201, in execute
          self.error_handler.check_response(response)
        File "/home/travis/build/ccnmtl/worth2/ve/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 102, in check_response
          value = json.loads(value_json)
        File "/opt/python/2.7.9/lib/python2.7/json/__init__.py", line 338, in loads
          return _default_decoder.decode(s)
        File "/opt/python/2.7.9/lib/python2.7/json/decoder.py", line 366, in decode
          obj, end = self.raw_decode(s, idx=_w(s, 0).end())
        File "/opt/python/2.7.9/lib/python2.7/json/decoder.py", line 384, in raw_decode
          raise ValueError("No JSON object could be decoded")
      ValueError: No JSON object could be decoded
```
Here's the failing travis build:
https://travis-ci.org/ccnmtl/worth2/builds/102458416

I'm using the Firefox driver.

**Comment by josephwon0310** [comment]
I'm also having issues with this. Merely trying to dismiss an alert from clicking href tag, and it spits out ValueError: No JSON object could be decoded.

**Comment by lukeis** [comment]
the fix here is pending a new python release

**Comment by nikolas** [comment]
@josephwon0310 it looks like the fix for this is released in version 2.49.1: https://github.com/SeleniumHQ/selenium/blob/master/py/CHANGES#L3
